### ISSUE TITLE
Syntax Highlighting in Class is broken and appearance of second configuration file (T)

### ISSUE THREAD

**Issue by FullValueRider** [WaynePhillipsEA/twinbasic#277]
**Describe the bug**
Please refer to the attached project.

Two issues

1.  The syntax highlighting in the Types class is not working correctly
2. I appear to have acquired a second configuration file called 'T'

I was updating Modules to Predeclared Classes (to allow simpler access from a library) when things went awol.

[VBALib.zip](https://github.com/WaynePhillipsEA/twinbasic/files/6830054/VBALib.zip)

**Desktop (please complete the following information):**
 - OS: WIndows 10
 - twinBASIC compiler version v0.10.3429, 15th July 2021

**Comment by dmrvb** [comment]
I confirm that I am seeing something similar. 
I have created a project with several modules and classes (using F1 Twinbasic add a new class to this project) and
then pasted VB6 code into these modules and classes.
The highlighting failed badly in one class only.
As I fixed various problems the highlighting got better but still has small issues.
The code that I paste in appears to be what causes the problem.
If I create a new class and paste in the same code then that also fails.
If I create a simple new project with just the bad call then it works ok.
